app/controllers/concerns/goto_preview.rb

```diff
@@ -8,4 +8,5 @@ def param_form_key
   end
 
   def goto_preview? = params.dig(param_form_key, :goto_preview) == 'true'
+  def goto_provider? = params.dig(param_form_key, :goto_provider) == 'true'
```

Comment:
We need to go back to the new provider page in some cases where there's no provider setup. Previously it would go to the preview page of the course but with moving the provider into its own view, we need to go back to that view not the preview